### PR TITLE
7-cleanup-responsible-nodes

### PR DESCRIPTION
**Pull request by VHPL-UIS**

### CODE REVIEW

File: model/parts/types/graph.go [hunk 1]
```diff
@@ -34,10 +33,6 @@ type EdgeAttrs struct {
 	Threshold int
 }
 
-//func (g *Graph) FindResponsibleNodes(chunkId int) [4]int {
-//	return g.RespNodes[chunkId]
-//}
-
 func BinarySearchClosest(arr []NodeId, target int, n int) ([]NodeId, NodeId) {
```
Comment (leandernikolaus) [inline model/parts/types/graph.go:36]: This function `BinarySearchClosest` should probably also be removed.
I believe it is not used for anything but finding responsible nodes.

**Review by leandernikolaus — COMMENTED** [review]
Function: `PrecomputeRespNodes` should also be removed, and the corresponding config option.

File: model/parts/types/graph.go (hunk 1 above)
Comment (VHPL-UIS) [reply model/parts/types/graph.go]: Can you please check now.

**Review by leandernikolaus — APPROVED** [review]
This should be good.

File: model/parts/types/graph.go (hunk 1 above)
Comment (leandernikolaus) [reply model/parts/types/graph.go]: Nice, please merge now

**Pull request merged**